docs/questionSharing.md

```diff
@@ -0,0 +1,40 @@
+**NOTE** Question sharing is currently in beta. If your course is hosted on the official PrairieLearn server, you may request that question sharing be turned on for your course. If you administer your own PrairieLearn server, you should _not_ use question sharing at this time. In the future, question sharing will be supported across PrairieLearn servers, so turning on question sharing on unofficial servers at this point in time may lead to naming conflicts in the future.
+
+# Question Sharing
+
+In order to avoid instructors needing to copy question files in between courses, PrairieLearn provides a way for questions from one course to be used in assessments in other courses.
+
+## Sharing Names
+
+In order for another course to import questions from your course into their assessments, you must have chosen a _sharing name_ for your course that they will use as a prefix to your question IDs when importing them. This sharing name will be unique across all PrairieLearn instances and because it will be used in the JSON files for other courses, there will be no way to change the sharing name for your course once you have chosen it. It is recommended that you choose something short but descriptive. For example, if you're teaching a calculus course at a university that goes by the abbreviation 'XYZ', then you could choose the sharing name 'xyz-calculus'. Then other courses will import questions from your course with the syntax `@xyz-calculus/qid`.
```

Comment:
I'm just slightly wary of using the term "import" here. Maybe stick with "use", "consume", and similar words?

Reply:
I've switched it to 'use' for now.